internal/install/application_configuration.go

```diff
@@ -21,12 +21,14 @@ import (
 )
 
 const (
-	stackVersion715 = "7.15.0-SNAPSHOT"
-	stackVersion820 = "8.2.0-SNAPSHOT"
+	stackVersion715  = "7.15.0-SNAPSHOT"
+	stackVersion820  = "8.2.0-SNAPSHOT"
+	stackVersion8160 = "8.16.0-00000000-SNAPSHOT"
```

Comment:
Using `8.16.0-SNAPSHOT` here as version did not change the docker image used for `8.16.0-21bba6f5-SNAPSHOT`.

Reply:
Maybe we can add a comment mentioning this. Or set it to `8.15.999`.

I guess we didn't have the same problem with the snapshots above because we were not using tagged versions, right?

Reply:
Exactly, previously tests were using SNAPSHOT versions without the commit in the prerelease tag.

I'll add a comment :+1:

Reply:
Added comment in https://github.com/elastic/elastic-package/pull/2038/commits/094f898a6f5a8795cbe082c18846f6adc2386bc5